pkg/config/validate.go

```diff
@@ -472,3 +473,19 @@ func (dc *DeploymentConfig) getStringValue(inputReference interface{}) (string,
 	}
 	return "", fmt.Errorf("the value %s is not a deployment variable or was not defined", inputReference)
 }
+
+func (dc *DeploymentConfig) testApisEnabled(validator validatorConfig) error {
+	requiredInputs := []string{}
+	funcName := testApisEnabledName.String()
+
+	if validator.Validator != funcName {
+		return fmt.Errorf("passed wrong validator to %s implementation", funcName)
+	}
+
+	err := testInputList(validator.Validator, validator.Inputs, requiredInputs)
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
Add a TODO here to actually run the validator. 

My personal preference would have been to write the function first, but I like that you are splitting it up.

Reply:
Done on `HEAD` of PR